unix/w0vncserver/PipewirePixelBuffer.cxx

```diff
@@ -292,13 +292,22 @@ void onStreamParamChanged(void *_data, uint32_t id,
     SPA_POD_Id(SPA_META_VideoCrop), SPA_PARAM_META_size,
     SPA_POD_Int(sizeof(struct spa_meta_region)));
 
+  /* Damage information */
+  params[nParams++] = (spa_pod *)spa_pod_builder_add_object(
+    &b, SPA_TYPE_OBJECT_ParamMeta, SPA_PARAM_Meta, SPA_PARAM_META_type,
+    SPA_POD_Id(SPA_META_VideoDamage), SPA_PARAM_META_size,
+    SPA_POD_CHOICE_RANGE_Int(sizeof(struct spa_meta_region) * 16,
+                             sizeof(struct spa_meta_region) * 1,
+                             sizeof(struct spa_meta_region) * 16));
```

Comment:
Does this limit how much damage information we might get? If so, 16 entries seems low?